pink/include/pink_conn.h

```diff
@@ -118,7 +118,7 @@ class PinkConn : public std::enable_shared_from_this<PinkConn> {
   std::string ip_port_;
   bool is_reply_;
   bool is_writable_;
-  bool close_;
+  bool close_{false};
```

Comment:
Generally I think this should be done in the constructor for this object, `is_writable` and `flags_` also appear to start uninitialized.